tools/test/audio/.octaverc

```diff
@@ -0,0 +1,3 @@
+more off
+pkg load signal
+pkg load io
```

Comment:
Can you not hide it? `octaverc`

Reply:
You should not need this. If you don't have it your $HOME you can use `exist('OCTAVE_VERSION', 'builtin')` to run these octave specific commands in beginning of script.

Reply:
I couldn't use `exist('OCTAVE_VERSION', 'builtin')` properly. I dropped the .octaverc then add `pkg load signal io` in front of octave command. How about that?